public/version-aliases/go.json

```diff
@@ -18,6 +18,8 @@
   "1.8.x": "1.8.7",
   "1.9.4": "1.9.4",
   "1.9.x": "1.9.4",
+  "1.10": "1.10",
```

Comment:
Unfortunately, this one is not going to work, because we only see `1.1` as a value passed.

Reply:
Sorry, but I don't understand yet.  Do you mean that if we write following in .travis.yml

```
language: go
language: go
go:
  - 1.10
```

then travis-build recognize it as not version `1.10` but `1.1` ?

Reply:
Unfortunately, yes. Parser interprets it as a float number, not as a string, and 1.1, 1.10, and 1.100 are all the same numbers.
There is an ongoing work in gimme itself to recognize `.x` aliases. After it is done, and a new version of gimme is released, my understanding is that this file can be removed.
@BanzaiMan please correct me if I'm wrong.

Reply:
Forgot to mention how to fix the problem now:
```yaml
language: go
go:
  - "1.10"
```

Reply:
Thanks, I understood!

Is there anything I should do to solve this problem?

Reply:
I think we should specify a character string for version, like perl https://docs.travis-ci.com/user/languages/perl/  ...

Reply:
That won't work in Go's case, because `1.1` is a also valid version. In contrast, in the case of Perl (5.20) and Node.js (0.10), their numerical counterparts (Perl 5.2) and (Node.js 0.1) are invalid versions, so we can map these values to the ones that work.

Reply:
Yes, the Go documentation should use strings instead of floats.